Userland/Libraries/LibCore/System.cpp

```diff
@@ -1549,7 +1549,7 @@ ErrorOr<void> unlockpt(int fildes)
     return {};
 }
 
-ErrorOr<void> access(StringView pathname, int mode)
+ErrorOr<void> access(StringView pathname, int mode, int flags)
```

Comment:
I would rather see the flags added to a Core::System::faccessat so that we don't drop them on the floor on non-serenity systems. That said, I'm not sure of the availability of `faccessat` on other unixes.